decidim-comments/app/frontend/comments/comment.component.jsx

```diff
@@ -0,0 +1,43 @@
+import { propType }    from 'graphql-anywhere';
+import gql             from 'graphql-tag';
+
+import commentFragment from './comment.fragment.graphql'
+
+/**
+ * @returns {ReactComponent} - A single comment component with the author info and the comment's body
+ */
+const Comment = ({ comment: { author, body, createdAt } }) => (
+  <article className="comment">
+    <div className="comment__header">
+      <div className="author-data">
+        <div className="author-data__main">
+          <div className="author author--inline">
+            <a className="author__avatar">
+              <img alt="avatar" src="../../assets/images/demo-avatar.jpg" />
```

Comment:
We don't have avatars but the design feels broken if I don't include it. I think the image is useful here, feels like "lorem ipsum" data to me so I'm ok with it. I'm totally against using `react-avatar` right now because it looks a bit overkill to me (right now)